### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by owengaspard**
I edited the README to be more concise and easier to follow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
-# Getting Started with Color
+# What is Color?
 
 A single page application that can show you monotone color palettes with different shades (S and L).
-The base idea is to get the hue from the user and use that to create the palette. 
+The idea is that it will take in a color from the user and output a palette.
```
Comment (SscSPs) [inline README.md]: I think mentioning the "HUE" usage makes more sense, can you add a piece that mentions that hue is taken from the color and the palette is created using that HUE?

**Comment by SscSPs** [comment]
Hey @owengaspard Thanks for making these changes, just added one comment, can you address that? we can merge once its concluded

**Comment by owengaspard** [comment]
> Hey @owengaspard Thanks for making these changes, just added one comment, can you address that? we can merge once its concluded

Done! Please check the latest commit to make sure it's ok.

**Comment by SscSPs** [comment]
Awesome, looks great!

**Pull request merged**